src/lib/pickles/dlog_main.ml

```diff
@@ -567,23 +701,16 @@ struct
 
   let combined_evaluation (type b b_plus_19) b_plus_19 ~xi ~evaluation_point
       ((without_degree_bound : (_, b_plus_19) Vector.t), with_degree_bound)
-      ~h_minus_1 ~k_minus_1 =
+      ~max_quot_size =
     let open Field in
     Pcs_batch.combine_split_evaluations ~mul ~last:Array.last
       ~mul_and_add:(fun ~acc ~xi fx -> fx + (xi * acc))
       ~shifted_pow:
         (Pseudo.Degree_bound.shifted_pow ~crs_max_degree:Common.Max_degree.wrap)
       ~init:Fn.id ~evaluation_point ~xi
-      (Common.dlog_pcs_batch b_plus_19 ~h_minus_1 ~k_minus_1)
+      (Common.dlog_pcs_batch b_plus_19 ~max_quot_size)
```

Comment:
Nit: I think `b_plus_19` here should be renamed to `b_plus_8` for this to remain self-documenting.